cart/player.go

```diff
@@ -202,12 +207,8 @@ func (player *Player) animate(t int32) {
 }
 
 func (player *Player) incrementFrame() {
-	if !player.Dead {
-		player.Frame = (player.Frame + 1) % 5
-		player.Sprite.Id = PLAYER_MAIN_FRAME + player.Frame
-	} else {
-		player.Sprite.Id = PLAYER_DEAD_FRAME
-	}
+	player.Frame = (player.Frame + 1) % 5
+	player.Sprite.Id = PLAYER_MAIN_FRAME + player.Frame
```

Comment:
Won't this cause the play to come back to life, at least as far as the sprite is concerned?

Reply:
It works, but I don't know why

Reply:
Ahhh. OK. We set the game to over when the player dies and that stops updates.

Reply:
👍